scripts/auto-rebase/rebase_job_entrypoint.sh

```diff
@@ -18,14 +18,23 @@ release_amd64="$(oc get configmap/release-release-images-latest -o yaml \
 release_arm64="$(oc get configmap/release-release-images-arm64-latest -o yaml \
     | yq '.data."release-images-arm64-latest.yaml"' \
     | jq -r '.metadata.name')"
+# LVMS is not tracked in the OCP release image.  Instead, rely on the latest from the same X.Y stream as the release image.
+# Input:
+#   registry.ci.openshift.org/ocp/release:4.13.0-0.nightly-2023-01-17-152326
+# Output:
+#    "4.13"
+release_lvms=$( echo "$release_amd64" | sed -E 's,.*:(v?[0-9]+\.[0-9]+).*,\1,')
```

Comment:
I think, in CI, release_amd64 is already a tag:
```
++ oc get configmap/release-release-images-latest -o yaml
++ yq '.data."release-images-latest.yaml"'
++ jq -r .metadata.name
+ release_amd64=4.13.0-0.nightly-2023-01-27-165107
```
so I guess you'd need to drop `.*:` from regexp.
Also, Giga is right - we might just hardcode 4.12 for now